clang/test/Sema/attr-nolock.cpp

```diff
@@ -0,0 +1,78 @@
+// RUN: %clang_cc1 %s -ast-dump -fblocks | FileCheck %s
+// expected-no-diagnostics
```

Comment:
`expected-no-diagnostics` only makes sense if you also have a `-verify` run directive.